packages/integration-tests/src/setupTestEnv.ts

```diff
@@ -0,0 +1,3 @@
+export default function globalSetup() {
+  process.env.TZ = "UTC";
```

Comment:
For my new test, I'm setting a date manually in the DB using a string. Then, I'm setting it to the same value via `new Date()`. This normalizes the timezone across developer machines for the test suite.